dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04330WorkflowTaskAddLanguageIdColumn.java

```diff
@@ -0,0 +1,275 @@
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotHibernateException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Config;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * This upgrade task will add the column language_id to the workflow_task, set the default language id to all of current records
+ * and add an index for the language_id.
+ *
+ * @author jsanca
+ * @version 5.0
+ *
+ */
+public class Task04330WorkflowTaskAddLanguageIdColumn extends AbstractJDBCStartupTask {
+
+    private static final Map<DbType, String> selectLanguageIdColumnSQLMap   = map(
+            DbType.POSTGRESQL,   "SELECT language_id FROM workflow_task",
+            DbType.MYSQL,        "SELECT language_id FROM workflow_task",
+            DbType.ORACLE,       "SELECT language_id FROM workflow_task",
+            DbType.MSSQL,        "SELECT language_id FROM workflow_task"
+            );
+
+    private static final Map<DbType, String> addLanguageIdColumnSQLMap      = map(
+            DbType.POSTGRESQL,   "ALTER TABLE workflow_task ADD language_id INT8",
+            DbType.MYSQL,        "ALTER TABLE workflow_task ADD language_id bigint",
+            DbType.ORACLE,       "ALTER TABLE workflow_task ADD language_id number(19,0)",
+            DbType.MSSQL,        "ALTER TABLE workflow_task ADD language_id NUMERIC(19,0) null"
+    );
+
+    private static final Map<DbType, String> updateLanguageIdColumnSQLMap   = map(
+            DbType.POSTGRESQL,   "UPDATE workflow_task SET language_id = ?",
+            DbType.MYSQL,        "UPDATE workflow_task SET language_id = ?",
+            DbType.ORACLE,       "UPDATE workflow_task SET language_id = ?",
+            DbType.MSSQL,        "UPDATE workflow_task SET language_id = ?"
+    );
+
+    private static final Map<DbType, String> addLanguageIdIndexSQLMap       = map(
+            DbType.POSTGRESQL,   "create index idx_workflow_6 on workflow_task (language_id)",
+            DbType.MYSQL,        "create index idx_workflow_6 on workflow_task (language_id)",
+            DbType.ORACLE,       "create index idx_workflow_6 on workflow_task (language_id)",
+            DbType.MSSQL,        "create index idx_workflow_6 on workflow_task (language_id)"
+    );
+
+    private static final Map<DbType, String> addLanguageIdForeignKeySQLMap       = map(
+            DbType.POSTGRESQL,   "alter table workflow_task add constraint FK_workflow_task_language foreign key (language_id) references language(id)",
+            DbType.MYSQL,        "ALTER TABLE workflow_task ADD CONSTRAINT FK_workflow_task_language FOREIGN KEY (language_id) REFERENCES language(id)",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 152 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)